src/controller/ExampleOperationalCredentialsIssuer.cpp

```diff
@@ -22,9 +22,35 @@
 namespace chip {
 namespace Controller {
 
+constexpr const char kOperationalCredentialsIssuerKeypairStorage[] = "ExampleOpCredsCAKey";
+
 using namespace Credentials;
 using namespace Crypto;
 
+CHIP_ERROR ExampleOperationalCredentialsIssuer::Initialize(PersistentStorageDelegate & storage)
+{
+    Crypto::P256SerializedKeypair serializedKey;
+    uint16_t keySize = static_cast<uint16_t>(serializedKey.Capacity());
+
+    if (storage.SyncGetKeyValue(kOperationalCredentialsIssuerKeypairStorage, serializedKey, keySize) != CHIP_NO_ERROR)
+    {
+        // Storage doesn't have an existing keypair. Let's create one and add it to the storage.
```

Comment:
Would be important to ensure that the resulting root cert is also stored, not just the key pair, so that it's deterministically the same root cert between invokations. A public root cert should never change contents for a given key pair.

Reply:
Yes, the root cert is being stored by the CHIP DeviceController class in the persistent storage.